liquibase-standard/src/main/java/liquibase/snapshot/jvm/PrimaryKeySnapshotGenerator.java

```diff
@@ -58,7 +58,7 @@ protected DatabaseObject snapshotObject(DatabaseObject example, DatabaseSnapshot
 
                 String ascOrDesc = row.getString("ASC_OR_DESC");
                 Boolean descending = "D".equals(ascOrDesc) ? Boolean.TRUE : "A".equals(ascOrDesc) ? Boolean.FALSE : null;
-                boolean computed = descending != null && descending;
+                boolean computed = descending != null;
```

Comment:
This test seems wrong. Descending is being used to hold 3 values here: true, false or null. 
d = true; a = false; else null . Maybe should be changed to an` if ` for clarity

Reply:
Done.